scripts/exregional_run_prepstart.sh

```diff
@@ -411,7 +411,7 @@ else
     ncatted -a checksum,,d,, fv_core.res.nc
 
 # generate coupler.res with right date
-    if [ ${cycle_subtype} == "spinup" ]; then
+    if [ ${cycle_subtype} == "spinup" ] && [ ${DO_ENSINIT} == "TRUE" ] ; then
```

Comment:
Do you want to update the comment in line 413 to reflect the following if block is only for ENSINIT?

Reply:
Updated the comment at line 415. Thanks!